frontend/src/metabase/redux/embed/embed.ts

```diff
@@ -1,10 +1,20 @@
-import { type PayloadAction, createSlice } from "@reduxjs/toolkit";
+import {
+  type PayloadAction,
+  asyncThunkCreator,
+  buildCreateSlice,
+} from "@reduxjs/toolkit";
 import { compose, pick } from "underscore";
 
 import { DEFAULT_EMBEDDING_ENTITY_TYPES } from "metabase/embedding-sdk/store";
 import { parseSearchOptions } from "metabase/lib/browser";
 import type { InteractiveEmbeddingOptions } from "metabase-types/store";
 
+import { setEntityTypes } from "../embedding-data-picker";
+
+export const createSlice = buildCreateSlice({
```

Comment:
This setup is required to use `create.asyncThunk`. [See the doc here.](https://redux-toolkit.js.org/api/createSlice#createasyncthunk)